tensorflow/core/kernels/depthwise_conv_op.cc

```diff
@@ -38,10 +38,14 @@ limitations under the License.
 #include "tensorflow/core/util/use_cudnn.h"
 #include "tensorflow/core/util/work_sharder.h"
 
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
```

Comment:
@jerryyin is `#if GOOGLE_CUDA` below a typo?

Reply:
@dagamyank Just for clarification, are you talking about line 43 being a typo (redundant)? Line 43 is necessary such that cuda related header is only pulled in under cuda environment. If you are talking about line 41, it is necessary because the include below only pull in GPU related header when necessary. 

Let me know if you mean something else.